content/nginx/admin-guide/installing-nginx/installing-nginx-open-source.md

```diff
@@ -9,548 +9,641 @@ type:
 - how-to
 ---
 
-This article explains how to install NGINX Open Source.
+This article explains how to install NGINX Open Source on various operating systems, including an overview of existing NGINX Open Source versions, installation types and methods, modules included in the default package and dynamic modules packages, and the basics of compiling nginx from the source code.
 
-<span id="stable_vs_mainline"></span>
-## Choosing Between a Stable or a Mainline Version
+## Stable and mainline versions {#stable_vs_mainline}
 
-NGINX Open Source is available in two versions:
+NGINX Open Source is available in two versions: Mainline and Stable. The source code and release notes for both versions are available from [nginx download page](https://nginx.org/en/download.html).
 
-- **Mainline** – Includes the latest features and bug fixes and is always up to date. It is reliable, but it may include some experimental modules, and it may also have some number of new bugs.
-- **Stable**  – Doesn’t include all of the latest features, but has critical bug fixes that are always backported to the mainline version. We recommend the stable version for production servers.
+### Mainline version
 
-<span id="compile_vs_package"></span>
-## Choosing Between a Prebuilt Package and Compiling from Source
+Mainline version (also Mainline release, Mainline branch) is the latest development version, updated approximately every 1 or 2 months, includes the latest features, bug fixes and security fixes. This version is recommended for production unless your organization has strict requirements for stability, in which case the stable version is the better choice. The Mainline version is always odd-numbered, for example, 1.*27*.X.
```

Comment:
Nitpick: Even though I see what you've bolded, I see "odd-number", and I think, "Oh, OK, the first 1 is an odd number." I wonder if you can describe it differently, something like:

```suggestion
Mainline version (also Mainline release, Mainline branch) is the latest development version, updated approximately every 1 or 2 months, includes the latest features, bug fixes and security fixes. This version is recommended for production unless your organization has strict requirements for stability, in which case the stable version is the better choice. The Mainline version always has an odd middle number, for example, 1.*27*.X.
```

I can visualize similar options such as "odd minor version number" or "odd second number"